spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Post', type: :request do
+  describe 'Get index route of users controller' do
+    it 'returns http success' do
+      get '/users'
+      expect(response).to have_http_status(:success)
+    end
+
+    describe 'Rendering the show page for the show routes' do
+      before :each do
+        get users_path(1)
+      end
+
+      it 'Returns success status for show routes' do
+        expect(response).to have_http_status(:success)
+      end
+
+      it 'Returns or renders the show template' do
+        expect(response).to render_template(:index)
+      end
```

Comment:
- [ ] Great job so far 👏🏻 however as requested by the previous reviewer:
> Post Spec file should include tests for both `show` and `index` that ensures the correct view template is rendered.
> `Example : expect(response).to render_template(:show)`

- Actually, you're missing the test that ensures the correct view `show` template is rendered. Please add this ✍🏻

Reply:
- [ ] As requested by the previous reviewers, please include a test to ensure that the correct template `show` is being rendered.